rc/targets.go

```diff
@@ -97,12 +97,13 @@ func NewClient(atcURL string, insecure bool) concourse.Client {
 	}
 
 	var transport http.RoundTripper
-
+	
 	transport = &http.Transport{
 		TLSClientConfig: tlsConfig,
 		Dial: (&net.Dialer{
 			Timeout: 10 * time.Second,
 		}).Dial,
+		Proxy: http.ProxyFromEnvironment,
```

Comment:
I set the proxy according to the default transport client, see: https://golang.org/src/net/http/transport.go#L34 (line 34)